scripts/sample-data.js

```diff
@@ -0,0 +1,145 @@
+
+const NUMBER_OF_LISTINGS = parseInt(process.argv[2] || '30')
+const NUMBER_OF_OFFERS = parseInt(process.argv[3] || NUMBER_OF_LISTINGS / 2)
+
+const NAMES = [
+  'Stan',
+  'Micah',
+  'Tyler',
+  'Josh',
+  'Daniel',
+  'Matthew',
+  'Kay',
+  'Пётр',
+  '계룡/鷄龍',
+  'Urist'
+]
+const CATEGORIES = ['Housing', 'Collectibles', 'For Sale', 'Services']
+const wordsFilePath = path.join(__dirname, "sample-words.json")
+const WORDLIST = JSON.parse(fs.readFileSync(wordsFilePath))
+
+function randomWord() {
+  return WORDLIST[Math.floor(Math.random() * WORDLIST.length)]
+}
+
+function randomTitleWord() {
+  const word = randomWord()
+  return word.charAt(0).toUpperCase() + word.slice(1)
+}
+
+/*
+ * Setup
+ */
+
+const web3Provider = new Web3.providers.HttpProvider('http://localhost:8545')
+const web3 = new Web3(web3Provider)
+const ipfsUrl = urllib.parse('http://origin-js:8080')
+const o = new Origin({
+  ipfsDomain: ipfsUrl.hostname,
+  ipfsGatewayProtocol: ipfsUrl.protocol.replace(':', ''),
+  ipfsGatewayPort: ipfsUrl.port,
+  web3
+})
+
+async function createSampleData() {
+  /*
+    * Create User Profiles 
+    */
+  const accounts = await web3.eth.getAccounts()
+  const users = []
+  for (const i in accounts) {
+    const firstName = NAMES[i]
+    const lastName = randomWord().toUpperCase()[0] + '.'
+    const account = accounts[i]
+    console.log(
+      chalk`⬢  Creating user {bold.hex('#09f4a6') ${firstName} ${lastName}} ${account}`
+    )
+    o.contractService.web3.eth.defaultAccount = account
+    if (Math.random() < 0.66) {
+      console.log('    ...with profile')
+      await o.users.set({
+        profile: {
+          firstName: firstName,
+          lastName: lastName
+        }
+      })
+    }
+    users.push(await o.users.get(account))
+  }
+
+  /*
+    * Create Listings 
+    */
+  const listings = []
+  for (let i = 0; i < NUMBER_OF_LISTINGS; i++) {
+    const brand = randomTitleWord()
+      .toUpperCase()
+      .replace(/[AEIOU]/, '')
+    const listingName = `${brand} ${randomTitleWord()}`
+    const user_i = Math.floor(Math.random() * users.length)
+    const user = users[user_i]
+    const userFirst = NAMES[user_i]
+    o.contractService.web3.eth.defaultAccount = user.address
+    const price = Math.floor(Math.pow(Math.random() * 10, 3.0)) / 100.0
+    const description = `${randomTitleWord()} is ${randomWord()}, ${randomWord()} ${randomWord()}.`
+    console.log(
+      chalk` ⬢  Creating listing {bold.hex('#d408f4') ${listingName}} from {bold.hex('#09f4a6') ${userFirst}}`
+    )
+    const newListing = await o.marketplace.createListing({
+      schemaVersion: '1.0.0',
+      listingType: 'unit',
+      title: listingName,
+      category: CATEGORIES[Math.floor(Math.random() * CATEGORIES.length)],
+      subCategory: CATEGORIES[Math.floor(Math.random() * CATEGORIES.length)],
+      language: 'en-US',
+      description: description,
+      price: { currency: 'ETH', amount: price.toString() },
```

Comment:
Feature request (can totally be done as separate PR): let's add a commission.
Something like:
`commission: { currency: 'OGN', amount: <same random amount> }`
That will allow us to test that ranking boosts listings with higher commission.